test/show.jl

```diff
@@ -35,4 +38,7 @@ module TestShow
     show(io, A)
     A = DataFrames.RepeatedVector([1, 2, 3], 1, 5)
     show(io, A)
+
+    render(Juno.Editor(), df)
```

Comment:
The return value of this is a JSON representation designed to be consumed by the frontend; in other words it's an implementation detail that's not part of the API. I'm assuming you don't want the DataFrames tests to break if/when Juno's implementation changes.